src/core_modules/capture-core/components/ScopeSelector/hooks/useOrganizationUnit.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import { useMemo } from 'react';
+import { useDataQuery } from '@dhis2/app-runtime';
+
+export const useOrganizationUnit = () => {
+    const { loading, data, refetch } = useDataQuery(
+        useMemo(
+            () => ({
+                organisationUnits: {
+                    resource: 'organisationUnits',
+                    id: ({ variables: { ownerOrgUnit: id } }) => id,
+                    params: {
+                        fields: ['displayName'],
+                    },
+                },
+            }),
+            [],
+        ),
+        {
+            lazy: true,
+        },
+    );
+
+    const referchOrganizationUnit = (ownerOrgUnit: string) => {
```

Comment:
Typo: refe**r**chOrganizationUnit. 

NIT: Can we use organ**s**ation  instead of organi**z**ation for consistency? 😬